apps/researcher/src/components/add-object-list-form/actions.ts

```diff
@@ -0,0 +1,19 @@
+'use server';
+
+import {getCommunityById} from '@/lib/community';
+import {objectList} from '@colonial-collections/database';
+import {revalidatePath} from 'next/cache';
+
+interface ListItem {
+  communityId: string;
+  createdBy: string;
+  name: string;
+  description?: string;
+}
+export async function addList(listItem: ListItem) {
```

Comment:
About the names: the function suggests that a 'list' is going to be added, but the argument suggests that a 'list item' is going to be added. Harmonize the names?

Reply:
Good point. This function creates a list. I will rename `listItem` to `list`